webui/module/Api/src/Api/Controller/DotJobController.php

```
@@ -58,48 +58,29 @@ public function getList()
+                'c' => 'Copy Job',
+                'g' => 'Migration Job',
+                'O' => 'Always Incremental Consolidate Job',
+                'V' => 'Verify Job',
+            ];
+
+            if ($type !== "executable") {
+                $types['R'] = 'Restore Job';
+            }
+
+            $jobs = [];
+            foreach ($types as $type_char => $type_label) {
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$type_label'. (undefined)

(UnusedLocalVariable)